### PR TITLE
Added bypass for non-host game quitter

### PR DESCRIPTION
**Pull request by nathanduchesne**
If merged, this PR will not allow non-hosts to quit the game: Instead, they will be presented with a funny message that tells them to stay in the game!

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -120,5 +120,20 @@
...
+            >"Are you really quitting because you're losing?"</item>
+        <item
+            >"Hmmm you may have misclicked on the quit button."</item>
+        <item
+            >"Nice try bud, you're staying here."</item>
+        <item
+            >"I doubt you really want to leave."</item>
+        <item
+            >"Quitting is for losers, we don't do that here."</item>
+    </string-array>
```
Comment (VictorSchneuwly) [inline app/src/main/res/values/strings.xml:137]: I love that you can have different sentences every time you try to quit 😂

**Review by VictorSchneuwly — COMMENTED** [review]
I don't know if that would be an issue here but when the `lyricsPopup` was open and something else was happening in the background, the game had some unexpected behaviors. So I would suggest you to update (and rename) the method `closeLyricsPopup` to also close yours. Better be safe than sorry 😉

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (nathanduchesne) [reply app/src/main/res/values/strings.xml:137]: Thanks! We don't allow them to quit, we may as well make it fun ;)

**Review by VictorSchneuwly — APPROVED** [review]
You can remove the comments like I said below but other than that it's all good to me. 👍🏻

File: app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt [hunk 2]
```diff
@@ -112,7 +113,7 @@ open class GameActivity : AppCompatActivity(), GameView, Timer.Listener {
 
     override fun displaySong(songName: String) {
         // Close the lyrics popup if already open
```
[the remark this answers is not in the document]
Comment (nathanduchesne) [reply app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt]: Done!

[the remark this answers is not in the document]
Comment (nathanduchesne) [reply app/src/main/java/com/github/fribourgsdp/radio/game/GameActivity.kt]: Done!

**Pull request merged**